web/src/elements/forms/Form.ts

```diff
@@ -191,6 +241,9 @@ export abstract class Form<T> extends AKElement {
         return json as unknown as T;
     }
 
+    /**
+     * As far as anyone can remember, this isn't being used.
```

Comment:
I actually meant to fix this, this function should be used almost everywhere above in `serializeForm`, see https://github.com/goauthentik/authentik/pull/6806/commits/67bb50ac88f70a3b75845ccf036e7c76848e79d9

Reply:
(For context, with this wrapper we can set `name` of an `ak-form-element-horizontal` to `foo.bar` and it will be translated to `data['foo']['bar']`, which we need at least for the RBAC stuff)

Reply:
Oh!  Okay, that makes sense.